src/frontend/handlers.go

```diff
@@ -79,8 +80,20 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
-	//get env and render correct platform banner.
-	var env = os.Getenv("ENV_PLATFORM")
+	// Set ENV_PLATFORM (checks for GCP. otherwise uses env value.)
+	var env = "local"
```

Comment:
Right, if the user removes the env var entirely I wanted to avoid a glitchy frontend. I instead commented out the default env in the YAML, and set the local default in the code only.